iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/serviceclient/JobClientCommon.java

```diff
@@ -30,7 +31,6 @@
 /**
  * Integration E2E test for Job Client on the service client.
  */
-@Ignore //ignoring these tests until icm bug 77555728 is resolved
```

Comment:
Please do the same on the Android side JobClientITonAndroid.java

Reply:
sure thing